src/modules/metadata/actions.js

```diff
@@ -47,10 +36,15 @@ export const PROJECT_TABLE_DELETION = createFlowActionTypes("PROJECT_TABLE_DELET
 export const FETCH_INDIVIDUAL = createNetworkActionTypes("FETCH_INDIVIDUAL");
 export const FETCH_OVERVIEW_SUMMARY = createNetworkActionTypes("FETCH_OVERVIEW_SUMMARY");
 
+export const DELETE_DATASET_DATA_TYPE = createNetworkActionTypes("DELETE_DATASET_DATA_TYPE");
 
-const endProjectTableAddition = (project, table) => ({type: PROJECT_TABLE_ADDITION.END, project, table});
-const endProjectTableDeletion = (project, tableID) => ({type: PROJECT_TABLE_DELETION.END, project, tableID});
-
+export const deleteDatasetDataType = networkAction((datasetId, dataType) => (dispatch, getState) => ({
```

Comment:
for data types, i wonder if `clear` makes more sense as a verb?

Reply:
I agree